src/components/atoms/ProductImage.tsx

```diff
@@ -23,6 +27,11 @@ export const ProductImage = styled.img<{ size: 'thumbnail' | 'tile' | 'popup' |
                   : '8rem'};
     object-fit: cover;
     flex: 0 0 auto;
+    ${({ rounded, theme }) =>
+        rounded &&
```

Comment:
nie rób rounded bo albo masz rounded cały projekt albo border-radius globalny na 0 to niepotrzebny props